app/views/idv/in_person/ready_to_verify/show.html.erb

```diff
@@ -33,31 +33,129 @@
+    <h2 class="margin-top-0 margin-bottom-2"><%= t('in_person_proofing.headings.barcode_what_to_bring') %></h2>
+    <p><%= t('in_person_proofing.body.barcode.what_to_bring') %></p>
+
+    <%# Option 1: Bring a Real ID %>
+    <h3 class="margin-y-3"><%= t('in_person_proofing.process.eipp_bring_id.heading') %></h3>
+    <div class="grid-row">
+      <%= image_tag(
+            asset_url('idv/real-id.svg'),
+            width: 130,
+            height: 87,
+            alt: t('in_person_proofing.process.eipp_bring_id.image_alt_text'),
+            class: 'grid-col-auto margin-left-0 margin-bottom-2',
+            role: 'img',
+          ) %>
+      <p class='grid-col-auto margin-left-0 tablet:grid-col-fill tablet:margin-left-3'><%= t('in_person_proofing.process.eipp_bring_id.info') %></p>
+    </div>
+    <hr class="margin-y-1" />
+
+    <%# Option 2: Bring a standard State ID... %>
+    <h3 class="margin-y-3"><%= t('in_person_proofing.process.eipp_what_to_bring.heading') %></h3>
+    <p><%= t('in_person_proofing.process.eipp_what_to_bring.info') %></p>
+
+    <%# A. State ID + Passport %>
+    <div class="grid-row margin-top-3">
+      <%= image_tag(
+            asset_url('idv/state-id-and-passport.svg'),
+            width: 130,
+            height: 128,
+            alt: t('in_person_proofing.process.eipp_state_id_passport.image_alt_text'),
+            class: 'grid-col-auto margin-left-0 margin-bottom-4',
+            role: 'img',
+          ) %>
+      <div class="grid-col-auto margin-left-0 tablet:grid-col-fill tablet:margin-left-3">
+        <p class="text-bold margin-y-0"><%= t('in_person_proofing.process.eipp_state_id_passport.heading') %></p>
+        <p><%= t('in_person_proofing.process.eipp_state_id_passport.info') %></p>
+      </div>
+    </div>
+    <hr class="margin-y-1" />
+
+    <%# B. State ID + Military ID %>
+    <div class="grid-row margin-top-3">
+      <%= image_tag(
+            asset_url('idv/state-id-and-military-id.svg'),
+            width: 130,
+            height: 100,
+            alt: t('in_person_proofing.process.eipp_state_id_military_id.image_alt_text'),
+            class: 'grid-col-auto margin-left-0 margin-bottom-4',
+            role: 'img',
+          ) %>
+      <div class="grid-col-auto margin-left-0 tablet:grid-col-fill tablet:margin-left-3">
+        <p class="text-bold margin-y-0"><%= t('in_person_proofing.process.eipp_state_id_military_id.heading') %></p>
+        <p><%= t('in_person_proofing.process.eipp_state_id_military_id.info') %></p>
+      </div>   
+    </div>
+    <hr class="margin-y-1" />
+
+    <%# C. State ID + two supporting documents %>
+    <div class="grid-row margin-top-3">
+      <%= image_tag(
+            asset_url('idv/state-id-and-fair-evidence-documents.svg'),
+            width: 130,
+            height: 150,
+            alt: t('in_person_proofing.process.eipp_state_id_supporting_docs.image_alt_text'),
+            class: 'grid-col-auto margin-left-0 margin-bottom-4',
```

Comment:
I notice in the screenshots that these images aren't the same width and the text doesn't line up as a result. Should we be used a fixed column width size here instead and letting the image fill to that constraint?

![image](https://github.com/18F/identity-idp/assets/1779930/ccbc81d7-6ad2-4fc1-bc34-468f1ab5fc94)

Reply:
Thanks @aduth  I have a fix for it. We swapped out images and it goofed up the UI. I will push up a bunch of ux changes in about 5 minutes.